assets/helpers/url.js

```diff
@@ -92,6 +92,14 @@ function getBaseDomain(): Domain {
 
 }
 
+function getAbsoluteURL(path: string): string {
```

Comment:
Time to get my own back. Could you add some tests for this? 😁

Reply:
I am proud that I passed the behaviour to require tests  😄 🥇

Reply:
👍